tests/Unit/Models/SystemSettingsTest.php

```
@@ -239,6 +242,24 @@ public function getNormalizesUnsupportedFallbackAndQualityAiPolicy(): void
         $this->assertSame('gpt-5.5-mini', $settings['ai_quality_model']);
     }
 
+    #[Test]
+    public function getNormalizesAiResponseCachePolicy(): void
+    {
+        $db = $this->makeDb($this->settingsRow([
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)